components/frontend/src/widgets/Button.js

```diff
@@ -155,20 +195,15 @@ export function AddDropdownButton({ itemSubtypes, itemType, onClick, allItemSubt
                             placeholder={`Filter ${itemType} types`}
                             value={query}
                         />
-                        {allItemSubtypes?.length > 0 && (
-                            <FilterCheckbox
-                                label={`Select from all ${itemType} types`}
-                                filter={showUnsupportedItems}
-                                setFilter={setShowUnsupportedItems}
-                            />
-                        )}
-                        {usedItemSubtypeKeys?.length > 0 && (
-                            <FilterCheckbox
-                                label={`Hide ${itemType} types already used`}
-                                filter={hideUsedItems}
-                                setFilter={setHideUsedItems}
-                            />
-                        )}
+                        <FilterCheckboxes
+                            itemType={itemType}
+                            allowHidingUnsupportedItems={allItemSubtypes?.length > 0}
+                            showUnsupportedItems={showUnsupportedItems}
```

Comment:
Although the variable name is the same as the arg, wouldn't it be more explicit to include the word "filter" in the arg name?

Reply:
Oh, nvm. I only just saw that this is also a boolean